Automated macOS build failure

It seems that the automated macOS build action is consistently failing (on any branch) since Nov 16, with this error in the _Configure build system_ phase:
```
-- Checking for module 'gtkmm-3.0>=3.16'
--   Package 'sigc++-2.0', required by 'glibmm-2.4', not found
CMake Error at /usr/local/Cellar/cmake/3.18.4/share/cmake/Modules/FindPkgConfig.cmake:545 (message):
  A required package was not found
```
By looking at the _Install dependencies_ phase logs, i suspect this might be related to a version conflict between `sigc++` versions:

```
==> Installing dependencies for gtkmm3: libsigc++@2, glibmm, atkmm, cairomm and pangomm
==> Installing gtkmm3 dependency: libsigc++@2
==> Pouring libsigc++@2-2.10.4.catalina.bottle.tar.gz
🍺  /usr/local/Cellar/libsigc++@2/2.10.4: 44 files, 1.3MB
==> Installing gtkmm3 dependency: glibmm
```
... then, a few lines below ...
```
==> Installing dependencies for adwaita-icon-theme: librsvg
==> Installing adwaita-icon-theme dependency: librsvg
==> Pouring librsvg-2.50.1.catalina.bottle.tar.gz
==> /usr/local/opt/gdk-pixbuf/bin/gdk-pixbuf-query-loaders --update-cache
🍺  /usr/local/Cellar/librsvg/2.50.1: 48 files, 137.6MB
==> Installing adwaita-icon-theme
==> Pouring adwaita-icon-theme-3.38.0.catalina.bottle.tar.gz
🍺  /usr/local/Cellar/adwaita-icon-theme/3.38.0: 6,004 files, 22.7MB
==> Downloading https://homebrew.bintray.com/bottles/libsigc%2B%2B-3.0.4.catalina.bottle.tar.gz
==> Pouring libsigc++-3.0.4.catalina.bottle.tar.gz
Unlinking /usr/local/Cellar/libsigc++@2/2.10.4... 5 symlinks removed
🍺  /usr/local/Cellar/libsigc++/3.0.4: 51 files, 338.5KB
```

so, it looks like libsigc++-2 gets removed and replaced with libsigc++-3 which is required by another package.

Unfortunately, i don't have any macs available to test, or any idea on how homebrew works...

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -14,9 +14,9 @@ on:
       - created
 jobs:
   build:
-    runs-on: macos-10.15
+    runs-on: macos-latest
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
       - name: Install dependencies
         run: |
            date -u
@@ -86,7 +86,7 @@ jobs:
           echo "ARTIFACT_FILE=${ARTIFACT}" >> $GITHUB_ENV
           zsh -c 'echo "Bundled in $(printf "%0.2f" $(($[$(date +%s)-$(cat bundlestamp)]/$((60.))))) minutes"'
           exit
-      - uses: actions/upload-artifact@v1
+      - uses: actions/upload-artifact@v2
         with:
           name: ${{env.ARTIFACT_FILE}}
           path: ${{env.ARTIFACT_PATH}}

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -23,7 +23,7 @@ jobs:
            mkdir build
            date +%s > build/stamp
            brew uninstall --ignore-dependencies libtiff
-           brew install libtiff gtk+3 gtkmm3 gtk-mac-integration adwaita-icon-theme libsigc++ little-cms2 libiptcdata fftw lensfun expat pkgconfig libomp shared-mime-info | tee -a depslog
+           brew install libtiff gtk+3 gtkmm3 gtk-mac-integration adwaita-icon-theme libsigc++@2 little-cms2 libiptcdata fftw lensfun expat pkgconfig libomp shared-mime-info | tee -a depslog
            date -u
            echo "----====Pourage====----"
            cat depslog | grep Pouring

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -82,8 +82,8 @@ jobs:
           echo "=== artifact: ${ARTIFACT}"
           # defining environment variables for next step as per
           # https://github.com/actions/starter-workflows/issues/68
-          echo "::set-env name=ARTIFACT_PATH::${GITHUB_WORKSPACE}/build/${ARTIFACT}"
-          echo "::set-env name=ARTIFACT_FILE::${ARTIFACT}"
+          echo "ARTIFACT_PATH=${GITHUB_WORKSPACE}/build/${ARTIFACT}" >> $GITHUB_ENV
+          echo "ARTIFACT_FILE=${ARTIFACT}" >> $GITHUB_ENV
           zsh -c 'echo "Bundled in $(printf "%0.2f" $(($[$(date +%s)-$(cat bundlestamp)]/$((60.))))) minutes"'
           exit
       - uses: actions/upload-artifact@v1

